storage/StorageBlockLayout.proto

```diff
@@ -76,6 +77,12 @@ message CSBTreeIndexSubBlockDescription {
   }
 }
 
+message SMAIndexSubBlockDescription {
+  extend IndexSubBlockDescription {
+    repeated int32 indexed_attribute_id = 128;
```

Comment:
Assigning the field `128` does not follow the rule described in `IndexSubBlockDescription`.

See #94 for the fix.